domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientTest.java

```diff
@@ -321,73 +325,21 @@ public boolean isSSL() {
     }
 
     @Override
-    public void close() {
-
-    }
-
-    @Override
-    public PgConnection preparedQuery(String s, Handler<AsyncResult<RowSet<Row>>> handler) {
-      handler.handle(Future.failedFuture("not implemented"));
-      return this;
-    }
-
-    @Override
-    public <R> PgConnection preparedQuery(String s, Collector<Row, ?, R> collector, Handler<AsyncResult<SqlResult<R>>> handler) {
-      handler.handle(Future.failedFuture("not implemented"));
-      return this;
-    }
-
-    @Override
-    public PgConnection query(String s, Handler<AsyncResult<RowSet<Row>>> handler) {
-      if (s.startsWith("EXPLAIN") && failExplain) {
-        handler.handle(Future.failedFuture("failExplain"));
-      } else if (s.startsWith("COUNT ") && asyncResult.succeeded()) {
-        List<String> columnNames = new LinkedList<>();
-        columnNames.add("COUNT");
-        RowDesc rowDesc = new RowDesc(columnNames);
-        Row row = new RowImpl(rowDesc);
-        row.addInteger(asyncResult.result().size());
-        List<Row> rows = new LinkedList<>();
-        rows.add(row);
-        RowSet rowSet = new LocalRowSet(asyncResult.result().size()).withColumns(columnNames).withRows(rows);
-        handler.handle(Future.succeededFuture(rowSet));
-      } else {
-        handler.handle(asyncResult);
-      }
-      return this;
-    }
-
-    @Override
-    public <R> PgConnection query(String s, Collector<Row, ?, R> collector, Handler<AsyncResult<SqlResult<R>>> handler) {
-      handler.handle(Future.failedFuture("not implemented"));
-      return this;
-    }
-
-    @Override
-    public PgConnection preparedQuery(String s, Tuple tuple, Handler<AsyncResult<RowSet<Row>>> handler) {
-      handler.handle(Future.failedFuture("not implemented"));
-      return this;
-    }
-
-    @Override
-    public <R> PgConnection preparedQuery(String s, Tuple tuple, Collector<Row, ?, R> collector, Handler<AsyncResult<SqlResult<R>>> handler) {
-      handler.handle(Future.failedFuture("not implemented"));
-      return this;
+    public Query<RowSet<Row>> query(String s) {
+      return null;
     }
 
     @Override
-    public PgConnection preparedBatch(String s, List<Tuple> list, Handler<AsyncResult<RowSet<Row>>> handler) {
-      handler.handle(Future.failedFuture("not implemented"));
-      return this;
+    public PreparedQuery<RowSet<Row>> preparedQuery(String s) {
+      return null;
     }
 
     @Override
-    public <R> PgConnection preparedBatch(String s, List<Tuple> list, Collector<Row, ?, R> collector, Handler<AsyncResult<SqlResult<R>>> handler) {
-      handler.handle(Future.failedFuture("not implemented"));
-      return this;
+    public void close() {
     }
   }
 
+  @Ignore
```

Comment:
Can you explain the ignores?
For example `@Ignore("Needs more work to make the fake connection work again")`

Reply:
Back now (not ignored)

Reply:
Very good!